manifests/config.pp

```diff
@@ -43,6 +50,15 @@
     mode  => '0640',
   }
 
+  concat { $dns::logconfpath:
```

Comment:
What is the reason you need a separate log file for this instead of adding it to the main concat?

Reply:
This was primarily due to the number of things that can be logged with named.  If I turn on all the logging options, that file is 154 lines long.  I thought potentially putting all of that in the main config file would be too much.